### PR TITLE
Activity details fixes #149 #151

### LINKED ISSUE #149: Add a specific image to the top of Activity details
For each activity we should add a representing image to the top of the Activity description to make it more appealing. This image then should also be used in the Activities overview.

### PR DESCRIPTION
**Pull request by AlexLakatos**
This is how activities look now.
ToDo in a later PR:
- better icon images for activity detail pages
- localized versions
webvr camp activate mozilla

### CODE REVIEW

File: _layouts/activity-default.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+<html lang="{{ page.lang }}">
+
+  {% include head.html %}
+
+  <body>
+
+    {% include header.html %}
+
+    <div class="page-content">
+      <div class="wrapper activity-page">
```
Comment (MichaelKohler) [inline _layouts/activity-default.html]: from what I remember without checking, the activity-page class is the only thing that changes from the default layout. Could you check if there is any other way how we could inject that so we don't need to do a separate file?

File: _pages/community-participation-guidelines.md [hunk 2]
```diff
@@ -1,26 +1,26 @@
 ---
-layout: page
+layout: activity
 title:  "Community Participation Guidelines (Draft) Review"
+subtitle: "In like 2-3 hours"
```
Comment (MichaelKohler) [inline _pages/community-participation-guidelines.md]: can we get rid of the "like" in all activities?

Comment (MichaelKohler) [inline _pages/community-participation-guidelines.md]: also let's make the "In" lowercase

File: _pages/community-participation-guidelines.md [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
+image: "/assets/img/guidelines-post-header.png"
 permalink: /community-participation-guideline/
 lang: en
 ref: community-participation-guidelines
 parent: activities
 redirect_from: "/experiments/community-participation-guidelines/"
 ---
 
+<div class="col-md-3 pull-right" markdown="1">
 * TOC
```
Comment (MichaelKohler) [inline _pages/community-participation-guidelines.md]: let's get rid of the TOC, and add sections as discussed through Telegram. Also, let's add the "impact and metrics" on the right side (totally fine to not show on mobile)

File: _pages/community-participation-guidelines.md [hunk 4]
```diff
@@ -83,10 +83,10 @@ You can sign up for a call using calendly.  Please select one of:
 
 **OR...**
 
-* Join our #Mozillians Telegram & Twitter live chat on [March 9th](https://www.timeanddate.com/worldclock/fixedtime.html?msg=CPG+Telegram+%26+Twitter+Chat&iso=20170309T07&p1=1091&ah=1) ! 
+* Join our #Mozillians Telegram & Twitter live chat on [March 9th](https://www.timeanddate.com/worldclock/fixedtime.html?msg=CPG+Telegram+%26+Twitter+Chat&iso=20170309T07&p1=1091&ah=1) !
```
Comment (MichaelKohler) [inline _pages/community-participation-guidelines.md]: as you have several of those fixes, want to add an .editorconfig?

File: _pages/e10s-addons.md [hunk 5]
```diff
@@ -1,15 +1,21 @@
 ---
-layout: page
+layout: activity
 title:  "e10s and Featured Add-ons"
+subtitle: "In like no time because it's retired"
```
Comment (MichaelKohler) [inline _pages/e10s-addons.md]: I'd remove the subtitle completely for retired ones. Also, can we add a light red background to the text or something similar to better show that this is retired?

File: _sass/_layout.scss [hunk 6]
```diff
@@ -6,7 +6,6 @@
     min-height: 40px;
     background: $brand-color;
     height: 60px;
-
```
Comment (MichaelKohler) [inline _sass/_layout.scss]: I like to have a linebreak before comments, but I guess that's a matter of taste.

File: _sass/_layout.scss [hunk 7]
```diff
@@ -42,9 +41,11 @@
     margin-right: 23px;
     float: left;
 
-    &, &:visited {
+    &,
+    &:visited {
```
Comment (MichaelKohler) [inline _sass/_layout.scss]: thx!

File: _sass/_layout.scss [hunk 8]
```diff
@@ -331,6 +325,36 @@
         margin-top: 40px;
     }
 
+    .activity-page {
+      padding: 0 0 20px 0;
```
Comment (MichaelKohler) [inline _sass/_layout.scss]: could you change that whole block to have 4 spaces as you changed above? ;)

File: _sass/_layout.scss [hunk 9]
```diff
@@ -331,6 +325,36 @@
...
+      background-color: #DAE3E7;
```
Comment (MichaelKohler) [inline _sass/_layout.scss]: given the fact that we want to adopt to mozilla style colors in the future, could you please add this as a variable?

File: _sass/_layout.scss [hunk 10]
```diff
@@ -331,6 +325,36 @@
...
+
+        @media(max-width: 768px) {
+          .col-md-9 {
+            width: 100%;
+          }
+        }
+        .col-md-3 {
+          width: 24%;
+          margin-right: -1%;
+          background-color: #FFFFFF;
```
Comment (MichaelKohler) [inline _sass/_layout.scss]: given the fact that we want to adopt to mozilla style colors in the future, could you please add this as a variable?

File: _sass/_layout.scss [hunk 11]
```diff
@@ -498,69 +517,58 @@
 
 .post-header {
     margin-bottom: $spacing-unit;
+    padding: 30px 0;
+    background: #f5f5f5;
```
Comment (MichaelKohler) [inline _sass/_layout.scss]: given the fact that we want to adopt to mozilla style colors in the future, could you please add this as a variable?

File: _sass/_layout.scss [hunk 12]
```diff
@@ -573,27 +581,28 @@
...
+    ol,
+    ul {
         margin-left: 0;
         padding-left: 15px;
     }
 }
-
 /* Language list */
-.en:lang(en), .es:lang(es), .pt:lang(pt), .fr:lang(fr) {
+.en:lang(en),
```
Comment (MichaelKohler) [inline _sass/_layout.scss]: can we make this somehow less generic to how many languages we have?

File: _sass/_layout.scss [hunk 13]
```diff
@@ -676,22 +718,6 @@ div.facebook {
     margin-top: -20px;
 }
 
-#markdown-toc {
```
Comment (MichaelKohler) [inline _sass/_layout.scss]: This breaks the FAQ TOC, but I kind of like it better like that.. ;)

screen shot 2017-03-15 at 23 18 39

File: _sass/_layout.scss [hunk 14]
```diff
@@ -751,28 +778,4 @@ div.facebook {
         display: none;
     }
 }
-
 // Bootstrap Overwrites
-table.sortable > thead th:hover:not([data-defaultsort="disabled"]) {
```
Comment (MichaelKohler) [inline _sass/_layout.scss]: why is this block removed?

* This breaks the hover on the tables and we again have an unreadable color
* This breaks the menu on md devices

**Comment by MichaelKohler** [comment]
Now, let's talk responsive.

* It looks like the TOC is not hidden on smaller screens
screen shot 2017-03-15 at 23 28 20
* I would remove the 1% border around the white background on mobile
* The picture in the header might need some margin on the left (or even break it to its own line).. Feel free to use flexbox
screen shot 2017-03-15 at 23 30 25
* Let's add a ```text-align: justify;``` for paragraphs on mobile. I think the following would look better:
screen shot 2017-03-15 at 23 32 43
* The first few paragraphs get earlier line-breaks than the ones after them, this might be because of the TOC though
screen shot 2017-03-15 at 23 33 11

File: _layouts/activity-default.html (hunk 1 above)
Comment (AlexLakatos) [inline _layouts/activity-default.html]: Been there, tried stuff. Jekyll has no page information at that level. Couldn't find a better way.

File: _sass/_layout.scss (hunk 13 above)
Comment (AlexLakatos) [inline _sass/_layout.scss]: I know, me too 🐶

File: _sass/_layout.scss (hunk 14 above)
Comment (AlexLakatos) [inline _sass/_layout.scss]: messed up the merge conflict I guess

File: _pages/community-participation-guidelines.md (hunk 4 above)
Comment (AlexLakatos) [inline _pages/community-participation-guidelines.md]: don't think I have one, but you can add an issue for it.

File: _sass/_layout.scss (hunk 12 above)
Comment (AlexLakatos) [inline _sass/_layout.scss]: I have no idea

**Comment by AlexLakatos** [comment]
Addressed most of the review.
I have no idea what images to use, maybe @couci can help me here? I need 150x150 pixel images please.
I think localization should happen separately, this PR is getting big enough already.

webvr camp activate mozilla 1

File: _sass/_layout.scss (hunk 12 above)
Comment (Ryuno-Ki) [inline _sass/_layout.scss]: Maybe via CSS variables?

Check browser support, but it could be an enhancement.

File: _sass/_layout.scss [hunk 15]
```diff
@@ -751,28 +790,8 @@ div.facebook {
...
-  background-color: $brand-color-light;
-}
-
-@include media-query($on-laptop) {
```
Comment (MichaelKohler) [inline _sass/_layout.scss]: this here should be in the PR as well (see comment above). Otherwise the menu looks really weird on md devices

**Comment by MichaelKohler** [comment]
Thanks! This is on the right track. I think we need to fix the images before we merge. The rest is just some smaller nits:

* Looks like the border is still there at the left side of the boxes on mobile
screen shot 2017-03-16 at 21 26 53
* I can scroll a few pixels to the right for everything below 821px width
* Let's center the title of the activity as well as the social buttons on mobile
screen shot 2017-03-16 at 21 28 34
* See comment about md menu

**Comment by MichaelKohler** [comment]
P1: merge of this
P2: file a follow up issue for l10n work

**Comment by couci** [comment]
@AlexLakatos I am (finally) gathering your pictures. Could you indicate which ones need an alternative picture from what we have on the launch page?

**Comment by couci** [comment]
foxface_puzzle-600x585
Addons added

**Comment by couci** [comment]
A-frame ones here: https://drive.google.com/drive/u/0/folders/0B1WwYOKD6M7vWWFLOWpiclhGaVU (thanks to @Srushtika )

**Comment by couci** [comment]
screen shot 2017-04-10 at 13 23 57
Webcompatability

**Comment by couci** [comment]
For test pilot (retired) let's use the firefox one same for e-10s. As for the CPG let's use the m from mozilla new logo since there is not a specific brand for there
https://assets.mozilla.org/portal/assets/#asset/205

**Comment by MichaelKohler** [comment]
I've updated the layout for ES/PT as well. Also I've deleted FR completely, as there is no need to keep this up to date with EN as long as there are no translations.

r+ from me for all changes, this can be merged if Laka agrees with my changes and has done the image swaps.

**Pull request merged**